vm/src/vm/interpreter.rs

```diff
@@ -22,7 +22,7 @@ use std::sync::atomic::Ordering;
 /// });
 /// ```
 pub struct Interpreter {
-    vm: VirtualMachine,
+    pub vm: VirtualMachine,
```

Comment:
ditto here. What do you use from `vm` that requires it be public? Can we add an additional method/function to cover the usecase?